russian/russian_lang.php

```diff
@@ -1,1635 +1,1925 @@
-$lang['payments_table_date_heading']          = 'Дата';
-$lang['payments_table_amount_heading']        = 'Сумма';
-$lang['payments_table_client_heading']        = 'Клиент';
-$lang['payment_not_exists']                   = 'Платеж не существует';
-$lang['payment_edit_for_invoice']             = 'Оплата счета';
-$lang['payment_edit_amount_received']         = 'Полученная сумма';
-$lang['payment_edit_date']                    = 'Дата оплаты';
+$lang['payments_table_mode_heading'] = 'Способ оплаты';
+$lang['payments_table_date_heading'] = 'Дата';
+$lang['payments_table_amount_heading'] = 'Сумма';
+$lang['payments_table_client_heading'] = 'Клиент';
+$lang['payment_not_exists'] = 'Платеж не существует';
+
+$lang['payment_edit_for_invoice'] = 'Оплата счета';
+$lang['payment_edit_amount_received'] = 'Полученная сумма';
+$lang['payment_edit_date'] = 'Дата оплаты';
+
 # Knowledge Base
-$lang['kb_article_add_edit_subject']   = 'Тема';
-$lang['kb_article_add_edit_group']     = 'Группа';
-$lang['kb_string']                     = 'База знаний';
-$lang['kb_article']                    = 'Раздел';
-$lang['kb_article_lowercase']          = 'раздел';
-$lang['kb_article_new_article']        = 'Новый раздел';
-$lang['kb_article_disabled']           = 'Откл.';
-$lang['kb_article_description']        = 'Описание раздела';
-$lang['kb_no_articles_found']          = 'В базе знаний ничего нет';
-$lang['kb_dt_article_name']            = 'Название раздела';
-$lang['kb_dt_group_name']              = 'Группа';
-$lang['new_group']                     = 'Новая группа';
-$lang['kb_group_add_edit_name']        = 'Название группы';
+$lang['kb_article_add_edit_subject'] = 'Тема';
+$lang['kb_article_add_edit_group'] = 'Группа';
+$lang['kb_string'] = 'База знаний';
+$lang['kb_article'] = 'Статья';
+$lang['kb_article_lowercase'] = 'статья';
+$lang['kb_article_new_article'] = 'Новая статья';
+$lang['kb_article_disabled'] = 'Отключено';
+$lang['kb_article_description'] = 'Описание статьи';
+$lang['kb_no_articles_found'] = 'Статьи не найдены';
+$lang['kb_dt_article_name'] = 'Название статьи';
+$lang['kb_dt_group_name'] = 'Группа';
+$lang['new_group'] = 'Новая группа';
+$lang['kb_group_add_edit_name'] = 'Название группы';
 $lang['kb_group_add_edit_description'] = 'Краткое описание';
-$lang['kb_group_add_edit_disabled']    = 'Откл.';
-$lang['kb_group_add_edit_note']        = 'Примечание: все статьи в этой группе будут скрыты, если установлен флажок "Откл."';
-$lang['group_table_name_heading']      = 'Название';
-$lang['group_table_isactive_heading']  = 'Активно';
-$lang['kb_no_groups_found']            = 'Нет групп';
+$lang['kb_group_add_edit_disabled'] = 'Отключено';
+$lang['kb_group_add_edit_note'] = 'Примечание: Все статьи в этой группе будут скрыты, если отметить флажок "Откл."';
+$lang['group_table_name_heading'] = 'Название';
+$lang['group_table_isactive_heading'] = 'Активно';
+$lang['kb_no_groups_found'] = 'Группы не найдены';
+
+# Mail Lists
```

Comment:
There are language keys which are removed from the main language files and are extracted in separate modules like , mail lists, surveys, goals, menu setup and backup

In the translation they are included again, can you please re-check them?